assets/i18n/qqq.json

```
@@ -485,6 +485,7 @@
 	"I18N_PREFERENCES_PROFILE_PICTURE_DRAG": "Text displayed in the preferences page. - Text of the dialog shown to upload a profile picture when the user has uploaded an image that is too big and needs to be cropped.",
 	"I18N_PREFERENCES_PROFILE_PICTURE_ERROR": "Text displayed in the preferences page. - Error text of the dialog shown to upload a profile picture. This error is shown when the file uploaded by the user is not an image.",
 	"I18N_PREFERENCES_PROFILE_PICTURE_UPLOAD": "Text displayed in the preferences page. - Title of the dialog shown to upload a profile picture.",
+	"I18N_PREFERENCES_REMOVE_PROFILE_PICTURE_BUTTON": "Text displayed in the preferences page. - Text of the button that confirms the action of the dialog shown to remove a profile picture.",
```

This feels too verbose. Would this be better?
```suggestion
	"I18N_PREFERENCES_REMOVE_PROFILE_PICTURE_BUTTON": "Text displayed in the preferences page. - Text of the confirm button in the dialog shown to remove a profile picture.",
```

---

Hey @kevintab95, actually it's not a confirm button. It is a separate 'Remove profile picture' button. Anyways, I will make it less verbose. Thanks!

---

Ah OK, then dropping "confirm" would do?

---

Done! Thanks